portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,42 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/messages")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
+    /*messages.add("{\"name\": \"Joey\", \"height\": \"6\"}");
+    messages.add("{\"name\": \"Matthew\", \"height\": \"6\"}");
+    messages.add("{\"name\": \"Henny\", \"height\": \"6\"}");*/
+    messages.add("message1");
+    messages.add("message2");
+    messages.add("message3");
+  }
```

Comment:
Is this a const set of messages? If so, you could use ImmutableList and initialize the list directly:
static final ImmutableList<String> MESSAGES = ImmutableList.of("message1", "message2");